module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -783,10 +820,13 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             foreach (
                 $this->getPagedResults(
                     'items',
-                    '/item-storage/items',
+                    '/inventory/items-by-holdings-id',
                     $query
                 ) as $item
             ) {
+                if ($item->discoverySuppress ?? false) {
```

Comment:
Is it no longer possible to pre-filter because of the API change? Is this a possible performance concern?

Reply:
Yes, unfortunately the `/inventory/items-by-holdings-id` API does not support the discoverySuppress query parameter.  I can submit that as a FOLIO Jira if it's a concern.  

The post-filtering itself is not a performance concern, I think, because in practice I'm not sure how often there will be an unsuppressed instance record, with an unsuppressed holdings record, with a significant number of suppressed item records.  Even if that were the case, we immediately ignore any suppressed records after the API call, so the only question is whether the changed API call itself is a performance hit.  Eliminating the filter in the call itself might actually speed things up.

That said, the switch from an item-storage API to the higher-level inventory API does have a negative performance impact.  In my very ad-hoc testing it adds an average of maybe 100ms to the call, ranging from almost zero difference to ~200ms at worst.  So then yeah, if there are 5 holdings records on the instance, that could in theory add a full second to the retrieval.  Again I am not sure if this is a practical concern, and there are probably other advantages to having the full (not storage-only) item record.  But if needed I could add a config param loadFullItem and allow the code to work both ways for the (probably large) majority of institutions that won't use bound-with.